.github/workflows/python-check-autogen.yml

```diff
@@ -0,0 +1,60 @@
+name: Check python autogenerated code is in sync
+on:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "LICENSE*"
+      - "**.gitignore"
+      - "**.md"
+      - "**.txt"
+      - ".github/ISSUE_TEMPLATE/**"
+      - ".github/dependabot.yml"
+      - "docs/**"
+  pull_request:
+    paths:
+      - "clients/python/src/mr_openapi/**"
+      - "api/openapi/model-registry.yaml"
+
+jobs:
+  check-autogen:
```

Comment:
I would recommend moving this job here:

https://github.com/kubeflow/model-registry/blob/366243346e606e3104586d81f0f54880d9787e32/.github/workflows/python-tests.yml#L17

wdyt?

Reply:
Thanks, it makes perfect sense to include this in the Python testing workflow 🙏🏼